install.sh

```diff
@@ -49,35 +60,47 @@ fi
 echo "-------------------------"
 echo "${color_green}[#]${color_normal} ${color_orange}makedeb Installer${color_normal} ${color_green}[#]${color_normal}"
 echo "-------------------------"
-
 echo
+
+if [[ "${DEBIAN_FRONTEND}" == "noninteractive" || "${NONINTERACTIVE}" == "1" || "${CI}" == "1" ]]; then
```

Comment:
I think we should just check if `NONINTERACTIVE` and `CI` are set, regardless of value. At least with `CI` I know my CI systems sets `CI=true`, and just checking if it's set should be enough to know the script's being ran in a CI environment imo. What do you think?